<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arAlembicExport</class>
 <widget class="QWidget" name="arAlembicExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>171</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>413</width>
    <height>100</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>arAlembicExport</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../img/img.qrc">
    <normaloff>:/shelf/software/maya/shelf/shelf_alembicExport35.png</normaloff>:/shelf/software/maya/shelf/shelf_alembicExport35.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(31, 31, 31);</string>
  </property>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>66</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>export checked groups</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../../img/img.qrc">
     <normaloff>:/btn/btn/btnAlembic91.png</normaloff>:/btn/btn/btnAlembic91.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>91</width>
     <height>46</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHelp">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../../img/img.qrc">
     <normaloff>:/btn/btn/btnHelp48.png</normaloff>:/btn/btn/btnHelp48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtMsg">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>266</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ALEMBIC EXPORT | Selectiv</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblCam">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../../img/img.qrc">:/btn/btn/btnCamera48.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblAssets">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>64</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../../img/img.qrc">:/shelf/software/maya/shelf/shelf_default35.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReport">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../../img/img.qrc">
     <normaloff>:/btn/btn/btnReport48.png</normaloff>:/btn/btn/btnReport48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxCam">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CAM</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAssets">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>65</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ASSETS</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAssetsChar">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>bull</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAssetsProp">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>110</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>horse</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAssetsSet">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>arena</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAssetsFx">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>fx</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblLight">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../../img/img.qrc">:/shelf/software/maya/shelf/shelf_suggestion35.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxLight">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>LGT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxMerge">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>15</y>
     <width>86</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>merge all groups into one abc</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">cbxMerge:checked {color: black; background-color: green; border: none; }</string>
   </property>
   <property name="text">
    <string>merge</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenFolder">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>139</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../../img/img.qrc">
     <normaloff>:/btn/btn/btnFolder48.png</normaloff>:/btn/btn/btnFolder48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxSelected">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>65</y>
     <width>76</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>export selected objects</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>SELECTED</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtFrameEnd">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>14</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1210</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtFrameStart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>14</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1001</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblFrameRange">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>15</y>
     <width>8</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxHidden">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>15</y>
     <width>86</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>export hidden objects as visible</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>hidden</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblUser">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>User</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent'</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/placeholder/placeholder/000_user.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblErrorCount">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>135</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent'; 
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <zorder>lblFrameRange</zorder>
  <zorder>btnExport</zorder>
  <zorder>btnHelp</zorder>
  <zorder>edtMsg</zorder>
  <zorder>lblCam</zorder>
  <zorder>lblAssets</zorder>
  <zorder>btnReport</zorder>
  <zorder>cbxCam</zorder>
  <zorder>cbxAssets</zorder>
  <zorder>cbxAssetsChar</zorder>
  <zorder>cbxAssetsProp</zorder>
  <zorder>cbxAssetsSet</zorder>
  <zorder>cbxAssetsFx</zorder>
  <zorder>lblLight</zorder>
  <zorder>cbxLight</zorder>
  <zorder>cbxMerge</zorder>
  <zorder>btnOpenFolder</zorder>
  <zorder>cbxSelected</zorder>
  <zorder>edtFrameEnd</zorder>
  <zorder>edtFrameStart</zorder>
  <zorder>cbxHidden</zorder>
  <zorder>line</zorder>
  <zorder>lblUser</zorder>
  <zorder>lblErrorCount</zorder>
 </widget>
 <resources>
  <include location="../../../../img/img.qrc"/>
 </resources>
 <connections/>
</ui>
